Project.toml

```diff
@@ -27,7 +27,7 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 xrootdgo_jll = "9d84c17e-11f2-50ef-8cc9-e9701362097f"
 
 [compat]
-Arrow = "2 - 2.5, ^2.6.1"
+Arrow = "2 - 2.5"
```

Comment:
sorry for my ignorance, but I guess 2.6 has a bug or so?

Reply:
yeah, https://github.com/apache/arrow-julia/pull/459